common/test/acceptance/tests/helpers.py

```diff
@@ -225,6 +225,9 @@ def select_option(query, value):
         try:
             select = Select(query.first.results[0])
             select.select_by_visible_text(value)
+            # If the element is still present, focus out of the element
```

Comment:
Your comment mentions "if the element is still present", but you don't check for that. What happens if the element is not present?